packages/video_player/video_player_platform_interface/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
```

Comment:
This needs to be published to be used by others, so I'd suggest:

```suggestion
## 6.2.0
```

(It will also require a change to the `version` field of the `pubspec.yaml` of the package.